src/opera_disp_tms/create_tile_map.py

```diff
@@ -4,14 +4,13 @@
 import subprocess
 import tempfile
 from pathlib import Path
-from typing import List
 
 from osgeo import gdal, gdalconst, osr
 
 gdal.UseExceptions()
 
 
-def get_tile_extent(info: dict, output_folder: Path) -> None:
+def get_tile_extent(info: dict, output_folder: Path):
```

Comment:
functions without an explicit `return` will return `None`, so I prefer keeping the type annotation
```suggestion
def get_tile_extent(info: dict, output_folder: Path) -> None:
```

https://docs.python.org/3/tutorial/controlflow.html#defining-functions
> The [return](https://docs.python.org/3/reference/simple_stmts.html#return) statement returns with a value from a function. return without an expression argument returns None. Falling off the end of a function also returns None.